web/src/components/filtersform.js

```diff
@@ -276,15 +280,37 @@ class FiltersForm extends React.Component {
       {params.get('approvals') && this.props.showChangeParams ? ', approvals ' + params.get('approvals') : ''}
       {params.get('exclude_approvals') && this.props.showChangeParams ? ', excluded approvals named ' + params.get('exclude_approvals') : ''}
       {params.get('state') && this.props.showChangeParams ? ', change state ' + params.get('state') : ''}
+      {params.get("project_definition") ? ", change project_definition " + params.get("project_definition") : ""}
       {'.'}
     </b>
     return resume
   }
+  handleProjectDefState = (e) => {
+    const selected = e.target.attributes.value.value;
+    this.setState({ project_definition: selected });
+  };
+
+  renderProjectDefs = async () => {
+    let res = await getConfigProjectDefinitions(this.props.location.pathname);
```

Comment:
Using await here seems unnecessary. Perhaps adding another then would work better, e.g.:  `getConfigProjectDefinitions(..).then(res => this.setState(..)`